packages/one-app-bundler/webpack/createResolver.js

```diff
@@ -23,12 +23,13 @@ const {
 // lib had a module or browser field in addition to the main field in its
 // package.json.
 
-module.exports = function createResolver({ mainFields }) {
+module.exports = function createResolver({ mainFields, resolveToContext = false }) {
   const enhancedResolver = ResolverFactory.createResolver({
     fileSystem: new CachedInputFileSystem(new NodeJsInputFileSystem(), 4000),
     useSyncFileSystemCalls: true,
     extensions: ['.js', '.jsx', '.json'],
     mainFields,
+    resolveToContext,
```

Comment:
What is `resolveToContext` mean in this case? I looked at https://github.com/webpack/enhanced-resolve and it wasn't clear what they meant by context.

Reply:
Resolving to context would be just to the module instead of the specific file. e.g. `~/dir/node_modules/ajv` instead of `~/dir/node_modules/ajv/lib/ajv.js`

Reply:
Ah I see. Thanks!